Tests/PackageGraphTests/DependencyResolverTests.swift

```diff
@@ -1072,6 +1073,17 @@ private func XCTAssertEqual(
     }
 }
 
+private func assertMissingVersions(
+    _ result: DependencyResolver.Result,
+    constraints: [PackageContainerConstraint],
+    file: StaticString = #file, line: UInt = #line
+) {
+    if case .error(let error as DependencyResolverError) = result,
+        case .missingVersions(let versions) = error {
+        XCTAssertEqual(versions, constraints)
```

Comment:
The `file` and `line` parameters above should be passed into the `XCTAssertEqual` call here otherwise failures will show up inside the assertion function rather than at its callsite.

Reply:
Oops. Thanks! Fixed below.